### DISCUSSION TITLE
Xdrip+ functionality

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#3070] [General]
Xdrip+ is such a well written app and the support is always wonderful. There is ONLY ONE feature that it lacks. The ability to send calibrated BG values to Libreview website like juggluco does. I continue to use juggluco to send UNCALIBRATED values to Libreview to satisfy my doctors and insurance. However it would be so nice if xdrip+ could send the calibrated BG values to Libreview in the same way Juggluco does it. Then the bg values in libreview would be more accurate because they are calibrated by xdrip+.
This functionality is a definite PLUS in Juggluco app, but it is raw sensor values which are highly inaccurate that are sent. I for one would like to see xdrip+ have this functionality so the bg numbers would be Accurate AND satisfy my doctors and insurance co. If this is possible, then please research that possible functionality for all us diabetic users that would like to see more accurate bg values reported.

**Comment by Navid200** [comment]
> There is ONLY ONE feature that it lacks. The ability to send **calibrated** BG values to Libreview website like juggluco does.   

Reading that sentence, I take you want to send calibrated values from xDrip to Libreview.
<br/>
  
> I continue to use juggluco to send **UNCALIBRATED** values to Libreview 

Reading that sentence tells me that using juggluco, you are also sending uncalibrated values.  
Then, why are you using it if it is doing exactly what xDrip does?
<br/>

**Comment by Npavkov** [comment]
Does xdrip+ already have the capability to be setup to send CALIBRATED bg values to Libreview?
I think sending more accurate CALIBRATED values to Lireview is more logical and would provide doctors with more accurate readings. I use Libre2 sensors which are traditionally inaccurate on the LOW side. I do use Juggluco currently to send uncalibrated values to Libreview. if xdrip+ can send CALIBRATED bg values to Libreview, then The send to Libreview option in Juggluco can be turned off.

**Reply by Navid200** [reply]
I'm not the person who can make this change.  But, I do monitor these forums in order to ask the right person to address questions and suggestions.
Before I can do that, I try to clarify the request.  So far, I fail to see why you have chosen to mention juggluco at all.
And you are not helping clarify what you are saying.

This is what you have posted:

> There is ONLY ONE feature that it lacks. The ability to send calibrated BG values to Libreview website like juggluco does.

So, are you saying juggluco can send calibrated values to Libreview?  Because that's what the second sentence in your quotes text claims.

Would you please focus and help me understand what's going on?

**Comment by Npavkov** [comment]
No Juggluco does not send CALIBRATED bg values to Libreview, **ONLY UNCALIBRATED** values (because juggluco does not do ANY Calibrating of raw values, that I know Of.) 
What I meant by "The ability to send calibrated BG values to Libreview website like juggluco does." was that juggluco has a option in it's settings to send values to libreview (it is just a checkbox). if the box is checked then Juggluco asks for your details of the Libreview Account. BUT Juggluco still **ONLY SENDS UNCALIBRATED** bg values to Libreview. CALABRATED values (in xdrip+) would be MUCH MORE ACCURATE readings. Those should be sent to Libreview so they  are more accurate.

**Reply by Navid200** [reply]
I will label this and someone hopefully will respond.  But, I doubt it would be a good idea for xDrip to send readings to an official platform that belongs to the manufacturer.

**Comment by Npavkov** [comment]
Why NOT....Juggluco DOES.

**Reply by Navid200** [reply]
xDrip is not meant to be used for dosing decisions.  xDrip is not FDA approved.  
You can ask juggluco why they do what they do.
The last thing we (xDrip) need is to be tagged as careless programmers who endanger lives by a large company.
We have to be careful in the path we take not to step on any toes.

@psonnera

**Comment by Npavkov** [comment]
Ok, I understand that!!  I also understand that Abbott (that owns libreview website) is also very protective of their stuff. I just thought since it is easier to provide calibrate (more accurate ) data to the xdrip+ home screen that it would be more logical to provide more accurate data to lireview website. if it cant be done, I understand there could be legal implications, so maybe I am just dreaming here. I will just have to provide Nightscout data to my doctor ( for more accurate readings of my BG values), for now. Thanks for listening ramble on, but it sounds like you should just ignore my request.

**Reply by psonnera** [reply]
Juggluco sending data to LibreView is already a free ticket for DMCA or legal actions. If you want to use DIY tecnologies prefer DIY reporting programs. Tidepool and Nightscout. IMHO. Which sensor are you using? If it's a Libre 2 you just need to scan it at least once every 8 hours to have all data. If you use a Libre 3 you can setup xDrip+ as a web follower from LibreView.
